configs/twins/twins_alt-gvt-b_uperhead_8x2_512x512_160k_ade20k.py

```diff
@@ -0,0 +1,35 @@
+_base_ = ['twins_alt-gvt-s_uperhead_8x2_512x512_160k_ade20k.py']
+model = dict(
+    type='EncoderDecoder',
+    pretrained='pretrained/alt_gvt_base.pth',
+    backbone=dict(
+        type='ALTGVT',
+        embed_dims=[96, 192, 384, 768],
+        num_heads=[3, 6, 12, 24],
+        depths=[2, 2, 18, 2]),
+    decode_head=dict(in_channels=[96, 192, 384, 768]),
+    auxiliary_head=dict(in_channels=384))
+
+optimizer = dict(
+    _delete_=True,
+    type='AdamW',
+    lr=0.00006,
+    betas=(0.9, 0.999),
+    weight_decay=0.01,
+    paramwise_cfg=dict(custom_keys={
+        'pos_block': dict(decay_mult=0.),
+        'norm': dict(decay_mult=0.)
+    }))
+
+lr_config = dict(
+    _delete_=True,
+    policy='poly',
+    warmup='linear',
+    warmup_iters=1500,
+    warmup_ratio=1e-6,
+    power=1.0,
+    min_lr=0.0,
+    by_epoch=False)
+
+# By default, models are trained on 8 GPUs with 2 images per GPU
```

Comment:
Is this comment necessary? If 8 GPUs with 2 images per GPU is default setting, we may note it in README.md like [BiSeNetV2 config](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/bisenetv2) for directly announcement to users.